tests/unit/libbpf_test.cpp

```diff
@@ -470,6 +470,95 @@ TEST_CASE("good tail_call", "[libbpf]")
 
 TEST_CASE("bad tail_call", "[libbpf]") { _ebpf_test_tail_call("tail_call_bad.o", -EBPF_INVALID_ARGUMENT); }
 
+TEST_CASE("recusrvice tail calls", "[libbpf]")
```

Comment:
```suggestion
TEST_CASE("recusrvice tail calls", "[libbpf]")
```
recursive is misspelled.  However, this isn't really recursive since no callee tries to tail call to itself.
```suggestion
TEST_CASE("multiple tail calls", "[libbpf]")
```